src/MyOrleansBundle/Resources/views/residence.html.twig

```diff
@@ -11,477 +11,479 @@
+                <div class="row">
+                    <div class="col s12 center">
+                        <div class="citation-residence" id="consult">
+                            <p>
+                                {{ residence.accroche }}
+                            </p>
+
+                            {% if residence.eligibilitePinel == 1 %}
+                                <p class="pinel-residence">Elligible Loi Pinel</p>
+                            {% else %}
+                                <p class="pinel-residence">Non Elligible Loi Pinel</p>
+                            {% endif %}
+                        </div>
                     </div>
+                </div>
+            </div>
 
-                    <div class="input-field col l12 m12 s12">
-                        <i class="material-icons prefix">my_location</i>
-                        {{ form_label(form.adresse) }}
-                        {{ form_widget(form.adresse) }}
-                        {{ form_errors(form.adresse) }}
+            <div class="container">
+                <div class="col s12">
+                    <div class="dispo">
+                        <h3>Nombre de logements disponibles : {{ flatsDispo }} sur {{ residence.nbTotalLogements }}</h3>
+                        <h3 class="consulter-residence">Consultez nos biens disponibles</h3>
                     </div>
-
-                    <div class="input-field col l12 m12 s12">
-                        <p class="newsletter">Quel est l'objet de votre message?</p>
-                        <i class="material-icons prefix">info_outline</i>
-
-                        {{ form_widget(form.sujet) }}
-                        {{ form_errors(form.sujet) }}
+                </div>
+                <div class="row">
+                    <div class="col s12">
+                        <div class="accordeon-residence">
+                            <ul class="collapsible popout" data-collapsible="accordion">
+                                <li>
+                                    <div class="collapsible-header">{{ typeLogement[0].nom }}</div>
+                                    <div class="collapsible-body">
+                                        <table class="stripped">
+                                            <thead>
+                                            <tr>
+                                                <th>Référence</th>
+                                                <th>Type</th>
+                                                <th>Surface</th>
+                                                <th>Prix</th>
+                                                <th>Fiche</th>
+                                            </tr>
+                                            </thead>
+
+                                            <tbody>
+                                            {% for flat in flats %}
+                                                {% if flat.typeLogement.nom == typeLogement[0].nom %}
```

Comment:
non mais c'est quoi cette triple boucle for avec typeLogement[0] [1] et [2]
tu as compris l'intérêt de faire des boucles ?